### PR TITLE
fix(2fa): Send 'security settings changed' email with 2fa enroll

### PR DESCRIPTION
**Pull request by maheskett**

### CODE REVIEW

File: src/sentry/api/endpoints/user_authenticator_enroll.py [hunk 1]
```diff
@@ -195,7 +195,7 @@ def post(self, request, user, interface_id):
                 context={
                     'authenticator': interface.authenticator,
                 },
-                send_email=False,
+                send_email=True,
```
Comment (mattrobenolt) [inline src/sentry/api/endpoints/user_authenticator_enroll.py]: 🤔 I wonder why we didn't do this to begin with.

File: tests/sentry/api/endpoints/test_user_authenticator_details.py [hunk 2]
```diff
@@ -203,7 +214,10 @@ def test_recovery_codes_regenerate(self):
         resp = self.client.get(url)
         assert old_codes != resp.data['codes']
 
-    def test_delete(self):
+        assert email_log.info.call_count == 0
```
Comment (billyvg) [inline tests/sentry/api/endpoints/test_user_authenticator_details.py]: Wouldn't we want this action to send an email?

File: tests/sentry/api/endpoints/test_user_authenticator_details.py [hunk 3]
```diff
@@ -155,6 +163,8 @@ def test_u2f_remove_device(self):
         resp = self.client.delete(url)
         assert resp.status_code == 500
 
+        assert email_log.info.call_count == 0
```
Comment (billyvg) [inline tests/sentry/api/endpoints/test_user_authenticator_details.py]: Removing a device seems like it should generate an email as well

**Review by billyvg — APPROVED** [review]

File: tests/sentry/api/endpoints/test_user_authenticator_details.py (hunk 2 above)
Comment (maheskett) [reply tests/sentry/api/endpoints/test_user_authenticator_details.py]: Ya, I'll add it in a follow-up pr ^

File: tests/sentry/api/endpoints/test_user_authenticator_details.py (hunk 3 above)
Comment (maheskett) [reply tests/sentry/api/endpoints/test_user_authenticator_details.py]: Ya, I'll add it in a follow-up pr ^

**Pull request merged**